docs/user-guide/management/aggregation-cache.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Streaming Aggregation in OpenObserv
+description: Learn how Streaming Aggregation accelerates dashboards using aggregation cache in OpenObserve Enterprise.
+---
+This document helps you understand what Streaming Aggregation is and how it relates to aggregation cache in OpenObserve.
+
+## What is Streaming Aggregation
+Streaming Aggregation is an Enterprise setting in OpenObserve that enables aggregation cache.
+
+!!! note "Who can use it"
+    All Enterprise users. 
+
+!!! note "Where to find it"
+    To enable aggregation cache: 
+
+    1. Go to **Management > General Settings**.  
+    2. Turn on the **Enable Streaming Aggregation** toggle.
+    3. Select **Save**.
+
+## What is Aggregation Cache
+Aggregation cache accelerates aggregation queries by saving results for each time partition when a dashboard panel runs. When a panel later requests the same or overlapping time range with the same stream and filters, OpenObserve reuses the saved partitions and computes only the missing partitions. Results remain correct because the system combines partition results using safe math such as adding sums and counts before computing an average.
```

Comment:
Its not just for dash board panels, it works on logs as well. Basically, any place where aggregate sql queries are used.